tests/integration/records/serializers/test_bibtex.py

```diff
@@ -29,6 +29,36 @@ def test_bibtex(api_client, db, create_record_factory):
     assert expected_result == response_data
 
 
+def test_bibtex_returns_all_expected_fields(api_client, db, create_record_factory):
+    headers = {"Accept": "application/x-bibtex"}
+    data = {
+        "_collections": ["Literature"],
+        "authors": [
+            {"full_name": "Smith, John", "inspire_roles": ["editor"]},
+            {"full_name": "Rossi, Maria", "inspire_roles": ["author"]},
+        ],
+        "control_number": 637275237,
+        "titles": [{"title": "This is a title."}],
+        "document_type": ["conference paper"],
+        "texkeys": ["Smith:2019abc"],
+    }
+    record = create_record_factory(
+        "lit", data=data, with_indexing=True, with_validation=True
+    )
+    record_control_number = record.json["control_number"]
+
+    expected_status_code = 200
+    expected_result = '@inproceedings{Smith:2019abc,\n    author = "Rossi, Maria",\n    editor = "Smith, John",\n    booktitle = "This is a title.",\n    title = "This is a title."\n}\n'
```

Comment:
actually `booktitle` is wrong here (it's supposed to come from the linked book record), probably another regression when moving the serializer to the new instance (compare https://github.com/inspirehep/inspire-next/blob/ba9d162b67ede9006892fea4e7b106495b0e445c/inspirehep/modules/records/serializers/fields_export.py#L278-L281 and https://github.com/inspirehep/inspirehep/blob/2f411395ccf666f683da840fa35f03dee91606f3/inspirehep/records/marshmallow/literature/bibtex.py#L246)